src/lamplib/tasks/__init__.py

```diff
@@ -46,12 +47,16 @@ def compile_all(context, env):
     logging.info('Compiling: %s', ' '.join(compile_cmd))
     subprocess.run(compile_cmd, env=env)
 
-
 def install(context, env):
+    
     install_cmd = [context.BUILD_SYSTEM, '-C', 'build', 'install']
     logging.info('Running install: %s', ' '.join(install_cmd))
     subprocess.run(install_cmd, env=env)
 
+    # Install python scripts
+    python_cmd = ['python3', 'setup.py', 'develop']
+    subprocess.run(python_cmd, cwd='src/python')
```

Comment:
We have to use a subprocess call instead of using the .py file more natively because setuptools makes use of the program's CLI arguments.